packages/core/src/deadLetter.ts

```diff
@@ -0,0 +1,19 @@
+import { fromTransition } from './actors/transition.ts';
+import type { AnyEventObject } from './types.ts';
+
+export const deadLettersBehavior = fromTransition<
+  any,
+  {
+    type: 'deadLetter';
+    event: AnyEventObject;
+  },
+  any
+>((state, event) => {
+  if (event.type === 'deadLetter') {
+    state.push({
+      event: event.event
+    });
+  }
```

Comment:
This is basically a memory leak. Those dead letters are never cleared.

Reply:
in Akka dead letters are not part of the state, but there is a concept of an event stream that is available on the system, so u can subscribe to it to be notified about all events (dead letters included)